client/components/product-card/index.jsx

```diff
@@ -33,6 +33,7 @@ class ProductCard extends Component {
 		] ),
 		fullPrice: PropTypes.oneOfType( [ PropTypes.number, PropTypes.arrayOf( PropTypes.number ) ] ),
 		isCurrent: PropTypes.bool,
+		isIncludedInCurrentPlan: PropTypes.bool,
```

Comment:
It's a shame we needed a new prop, but I guess there aren't many other ways to go around it.

Reply:
I don't like it as well 😞

There are at least two ways to get away from this:
1. Replace `isCurrent` and `isIncludedInCurrentPlan` to a single prop like `isPartOf`. It would take one of the two values: `product` or `plan`. I think it'd be still a bit awkward...
2. Move the logic to the `ProductSelector` block. This way the whole `ProductCardAction` sub-component would be passed to the `ProductCard` and so the logic would be moved away from the UI component. [We're doing something similar when rendering the checkout button](https://github.com/Automattic/wp-calypso/blob/c199ce5be4cce6f4af7acc317e75e478a6175860/client/blocks/product-selector/index.jsx#L311-L319).

I wonder what's your opinion about those options? Do you see any other option?

Reply:
I was thinking of 2. and wasn't sure if won't complicate things too much. But in the same time, sounds good to try it, because that way we'll keep the `ProductCard` dumb enough. Wanna try with 2?

Reply:
Sure, I'm going to try it out now.